app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -164,13 +165,7 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
 
         // Must be after we set the content view
         if (isVisuallyComplete) {
-            rootContainer.doOnPreDraw {
-                // This delay is temporary. We are delaying 5 seconds until the performance
-                // team can locate the real point of visual completeness.
-                it.postDelayed({
-                    visualCompletenessQueue!!.ready()
-                }, delay)
-            }
+            setupRootContainerPreDrawDelay(rootContainer, WeakReference(this))
```

Comment:
nit: this name describes the implementation detail rather than what we're trying to achieve. That makes it redundant to the code and generally not helpful when scanning through `onCreate()`. "Huh, what's the root container? What pre draw delay? Does it delay drawing the root container? Why?"

Rather, if this was called `attachVisualCompletenessQueue`, its intention is clearer and the way we attach the queue (the implementation) can change without needing to rename the method.